mula/tests/integration/test_api.py

```diff
@@ -426,53 +430,58 @@ def test_get_tasks_value(self):
 
     def test_get_tasks_min_and_max_created_at(self):
         # Get tasks based on datetime, both min_created_at and max_created_at, should return 2 items
-        min_created_at = self.first_item_api.get("created_at")
-        max_created_at = self.second_item_api.get("created_at")
+        min_created_at = urllib.parse.quote(self.first_item_api.get("created_at"))
```

Comment:
I was still wondering if this wrap is really needed since the client used here is usually really good at quoting and encoding. What I for sure know is that if you're using the `params` keyword the query parameters will get encoded properly. I just tested this and it should also work if you pass the url only:
```python3
>>> import requests
>>> res = requests.get('https://google.com/?this="iasd"a"compleX#@query&param=2313X#~~~')
>>> res.url
'https://www.google.com/?this=%22iasd%22a%22compleX#@query&param=2313X%23~~~'
```

Reply:
Fix implemented